exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,17 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+let totalNumber = numberOfStudents + numberOfMentors;
+
+let quotientOfStudents = numberOfStudents / totalNumber;
+let quotientOfMentors = numberOfMentors / totalNumber;
+
+let productOfStudents = quotientOfStudents * 100;
```

Comment:
could we have added the " * 100"  to line 5 to save creating an extra variable or alternatively add it into the console.log on line 14 : something like this: math.round(quotientOfStudents*100) + "%"

Reply:
thanks Mandeep i was able to take out four lines of code